export.js

```diff
@@ -0,0 +1,161 @@
+function grabExcelValues(rows, artifact, objTemplate, customFieldRange) {
+    if (rows === null) {
+        getRows(artifact, objTemplate, customFieldRange);
+    }
+    else {
+        return Excel.run(function (context) {
+            let sheet = context.workbook.worksheets.getItem(toSheetName(artifact));
+            let inputRange = columnRanges[artifact] + rows;
+            let customRange = (customFieldRange[0] + 3) + ":" + (customFieldRange[1] + rows);
+            let inputValues = sheet.getRange(inputRange);
+            let customFields = sheet.getRange(customRange);
+            inputValues.load();
+            customFields.load();
+            return context.sync()
+                .then(function () {
+                    customFieldObjArr = customFields.values.map(customFieldObjCreate);
+                    ajaxExport(inputValues.values, artifact, objTemplate, customFieldObjArr);
+                })
+        });
+    }
+}
+
+function getRows(artifact, objTemplate, customFieldRange) {
+    return Excel.run(function (context) {
+        let sheet = context.workbook.worksheets.getItem(toSheetName(artifact));
+        let sheetRange = sheet.getUsedRange();
+        sheetRange.load();
+        return context.sync()
+            .then(function () {
+                grabExcelValues(sheetRange.values.length, artifact, objTemplate, customFieldRange);
+            })
+            .catch(function (error) {
+                console.log(error);
+            })
+    });
+}
+
+function ajaxExport(valueArray, artifact, objTemplate, customFieldObjArr) {
+    let objArray = buildobjects(valueArray, artifact, objTemplate);
+    for (let i in customFieldObjArr){
+        objArray[i].CustomProperties = customFieldObjArr[i];
+    }
+    postNew(objArray, artifact, 0);
+    console.log(objArray);
+}
+
+function buildobjects(valueArray, artifact, objTemplate) {
+    let objArray = [];
+    for (let i = 0; i < valueArray.length; i++) {
+        let j = 0;
+        for (let prop in objTemplate) {
+            //grabs the digit from Importance Name field for the id
+            if (prop == "ImportanceId") {
+                objTemplate[prop] = valueArray[i][j].charAt(0);
+            }
+            else if (prop == "ReleaseId"){
```

Comment:
can you use something like `requirements.fields.releaseId` instead of "ReleaseId" - so keep all actual values and strings in one place in an easy to update object